css/src/components/tabs.scss

```diff
@@ -0,0 +1,51 @@
+// Probably keep, will need some big changes.
+
+$tabs-link-color: $text !default;
+$tabs-link-hover-background-color: $primary-hover !default;
+$tabs-link-radius: $border-radius-rounded !important;
+$tabs-link-active-background-color: $primary-active !default;
+$tabs-link-active-color: $primary-invert !default;
+
+.tabs {
+	display: flex;
+	align-items: stretch;
+	justify-content: space-between;
+	width: 100%;
+	border-radius: $tabs-link-radius;
+	background-color: $body-background-accent;
+
+	.tab-list {
+		display: flex;
+		flex-grow: 1;
+		align-items: center;
+		justify-content: flex-start;
+	}
+
+	.tab-control {
+		display: flex;
+		flex-grow: 1;
+		align-items: center;
+		justify-content: center;
+		border: none;
+		background-color: transparent;
+		color: $tabs-link-color;
+		padding-block: 1rem;
+		cursor: pointer;
+
+		&:hover {
+			border-radius: $tabs-link-radius;
```

Comment:
don't think we want radius set inside hover or active states, leads to strange focus circle changes on tab selection

Reply:
also, dragging a mouse through the space between buttons yields some very erratic hover behavior